app/components/base/PageFeedback.tsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Button, styled, Typography } from "@mui/material";
+import {
+  type SvgIconComponent,
+  ThumbDownOffAltOutlined,
+  ThumbUpOffAltOutlined,
+} from "@mui/icons-material";
+import merge from "lodash.merge";
+import { Link } from "./Typography";
+
+const FeedbackButton = styled(Button, { name: "FeedbackButton" })(() => ({
+  color: "rgba(0, 0, 0, 0.50)",
```

Comment:
Could we add this as a variable to the `app/components/colors.ts` file? 

Here's a naming convention I've been using in other DevPortal redesign updates that might work here as well:
```
// app/components/colors.ts

// Overlay Colors
export const GREY_050 = "rgba(0, 0, 0, 0.50)";
```